.github/workflows/release-please.yml

```
@@ -16,7 +16,11 @@ jobs:
     steps:
       - name: Create release PR
         id: release
+<<<<<<< HEAD
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)